docs/getting-started.md

```diff
@@ -1,338 +1,244 @@
 
-💡 At any point, you can **check out our [getting-started.ts](./getting-started.ts) for a working example of the code presented in the following**.
+💡 At any point, you can **check out our [getting-started.ts](./getting-started.ts) for a working example of the code presented here**.
 
-### 1.2 Initializing the KILT SDK
+### 1.2 Set up the crypto and connect to a KILT node
 
-When using the SDK, there are two things you'll always want to do before anything else:
+When using the SDK, there are two tasks that must performed before everything else:
 
 1. Initialize cryptographic dependencies.
-   If you don't do this, certain operations like account generation could fail with the notice that "the WASM interface has not been initialized".
-2. Set essential configurations, most importantly the endpoint of the KILT node to which you'll want to connect for actions that read or write to blockchain state.
-   These operations would throw an error if called before an endpoint has been set.
+   If this step is skipped, certain operations like account generation using some algorithms, e.g., Sr25519, could fail with the error "the WASM interface has not been initialized".
+2. Set essential configurations, most importantly the address of the KILT node to connect to for anything that interacts with the KILT blockchain.
+   These operations will throw an error if called before a connection with a KILT node has been established.
 
-To keep things simple, we grouped these two steps in a function you can call first thing, before any other code that used the KILT SDK.
+DID operations require the presence of a keystore object that implements the [Keystore interface](../packages/types/src/Keystore.ts).
+For the sake of simplicity, the SDK provides a [demo keystore implementation](../packages/did/src/DemoKeystore/DemoKeystore.ts) which can be used to generate key pairs that are kept unencrypted in memory and disappear at the end of the program execution.
+
+**Using the demo keystore in production is highly discouraged as all the keys are kept in the memory and easily retrievable by malicious actors. For an example on how to write your own keystore, take a look at the [Sporran wallet implementation](https://github.com/BTE-Trusted-Entity/sporran-extension).**
 
 ```typescript
 import * as Kilt from '@kiltprotocol/sdk-js'
 
-await Kilt.init({ address: YOUR_CHAIN_ADDRESS })
-```
-
-Again, this is asynchronous, so be sure to wrap this in an `async` function as described above.
-Add this line to the `async` functions wrapping the examples below if you run them individually.
-
-### 1.2. Connect to a Chain
-
-Either explicitly:
+/* Set up the crypto */
 
-```typescript
-Kilt.config({ address: YOUR_CHAIN_ADDRESS })
-await Kilt.connect()
-```
-
-Or by setting a default address in the configuration, connecting implicitly.
+// Keyring is required to generate KILT accounts.
+const keyring = new Kilt.Utils.Keyring({
+  ss58Format: 38,
+  type: 'ed25519',
```

Comment:
We should have a comment here about the `sr25519` requiring the `cryptoWaitReady`.